### DISCUSSION TITLE
Is there a Keyboard Command for the mute button?

### DISCUSSION THREAD

**Discussion by JaminShanti** [jamulussoftware/jamulus#2210] [General]
Is there a Keyboard Command for the mute button?
Or making it a button like disconnect.

**Comment by henkdegroot** [comment]
Are you referring to the "Mute Myself" option? If it is the last "selected" button (clicked), you can repeat the action by hitting the spacebar.

**Reply by JaminShanti** [reply]
That's a work around.  Any plans to move away from the check box which is very had to hit while playing. Or add a keyboard command.

**Reply by henkdegroot** [reply]
Not sure if it works for all OS's, but the ALT+M keyboard shortcut already exist as well. I am not aware of any plans for adding other keyboard commands/shortcuts.

**Comment by pcar75** [comment]
Bonjour,
Keyboard shortcuts makes great sense to reduce the time my hands are not on my instrument.  I will hitchhike on the thought in the Ideas category, #xxxx .

**Comment by henkdegroot** [comment]
Depending on the OS being used, also see this really interesting topic: [Keyboard shortcuts in Jamulus](https://sourceforge.net/p/llcon/discussion/533517/thread/f3252db0f5/)
